src/appmetrics.cpp

```diff
@@ -184,18 +189,24 @@ static std::string fileJoin(const std::string& path, const std::string& filename
 }
 
 static std::string* getModuleDir(Local<Object> module) {
-    std::string moduleFilename(toStdString(module->Get(Nan::New<String>(asciiString("filename")).ToLocalChecked())->ToString()));
+    Local<String> filenameKey = Nan::New<String>(asciiString("filename")).ToLocalChecked();
+    Local<String> filename = Nan::To<String>(module->Get(filenameKey)).ToLocalChecked();
+    std::string moduleFilename(toStdString(filename));
     return new std::string(portDirname(moduleFilename));
 }
 
 static Local<Object> getProcessObject() {
-    return Nan::GetCurrentContext()->Global()->Get(Nan::New<String>(asciiString("process")).ToLocalChecked())->ToObject();
+    Local<String> process = Nan::New<String>(asciiString("process")).ToLocalChecked();
+    return Nan::To<Object>(
+        Nan::GetCurrentContext()->Global()->Get(process)
+    ).ToLocalChecked();
 }
 
 static std::string* findApplicationDir() {
-    Local<Value> mainModule = getProcessObject()->Get(Nan::New<String>(asciiString("mainModule")).ToLocalChecked());
+    Local<String> mainModuleString = Nan::New<String>(asciiString("mainModule")).ToLocalChecked();
+    Local<Value> mainModule = getProcessObject()->Get(mainModuleString);
```

Comment:
```suggestion
    Local<Value> mainModule = Nan::Get(getProcessObject(), mainModuleString);
```

[`Nan::Get`](https://github.com/nodejs/nan/blob/master/doc/maybe_types.md#api_nan_get)